source/features/view-markdown-source.tsx

```diff
@@ -78,12 +78,12 @@ async function init(): Promise<void> {
 	delegate(document, '.rgh-md-source:not(.selected)', 'click', showSource);
 	delegate(document, '.rgh-md-rendered:not(.selected)', 'click', showRendered);
 
-	select('.repository-content .Box-header .d-flex')!.prepend(
+	select('.repository-content > .Box:nth-of-type(3) .Box-header .d-flex')!.prepend(
```

Comment:
We should try to preserve compatibility with the old design to enable GHE support. 

In this case you can use:

``` js 
const toolbar = select(new selector) ?? selector(old selector)!;
toolbar.prepend( JSX )

Reply:
The same applies to most of your other PRs

Reply:
I see, thanks for the reminder!